src/components/layers/JSONEditor.jsx

```diff
@@ -145,16 +154,22 @@ class JSONEditor extends React.Component {
   }
 
   render() {
+    const {showMessage} = this.state;
     const style = {};
     if (this.props.maxHeight) {
       style.maxHeight = this.props.maxHeight;
     }
 
-    return <div
-      className={classnames("codemirror-container", this.props.className)}
-      ref={(el) => this._el = el}
-      style={style}
-    />
+    return <div className="JSONEditor" onPointerDown={this.onPointerDown} aria-hidden="true">
+      <div class={classnames("JSONEditor__message", {"JSONEditor__message--on": showMessage})}>
```

Comment:
Looks like the failing tests could be fixed with:
```suggestion
      <div className={classnames("JSONEditor__message", {"JSONEditor__message--on": showMessage})}>
```

Reply:
I just realised that was a 'Suggested change' and I could have fixed that from the UI rather than my editor. Thats cool, didn't know you could do that 🤦‍♀️